scripts/globals/roe.lua

```diff
@@ -166,6 +166,8 @@ local defaults = {
                                 --        "weekly" - Weekly record.
                                 --        "unity"  - Weekly reset, but doesn't reset progress, only completion.
                                 --        "retro"  - Can be claimed retroactively. Calls check on taking record.
+                                --        "hidden" - Special internal-use record used only as a client-flag to unlock others.
```

Comment:
Hidden Records are handled client-side with no impact to core.  Is there a reason for separating this into an additional bitfield?

Reply:
Indeed, you are correct on all counts. However, those records shouldn't be counted by The function which counts how many records a player completed.